processor/processor.go

```diff
@@ -2354,6 +2355,7 @@ func (proc *Handle) transformSrcDest(
 			errorsPerDestID: procErrorJobsByDestID,
 			reportMetrics:   reportMetrics,
 			routerDestIDs:   routerDestIDs,
+			droppedJobs:     droppedJobs,
```

Comment:
didn't we have a test for dropped jobs?

Reply:
missed this one😓
will post some more test cases soon.